modules/ob-creating-a-buildah-build.adoc

```diff
@@ -12,11 +12,14 @@ You can create a `buildah` build and push the created image to the target regist
 
 * You have installed the {builds-operator} on the {ocp-product-title} cluster.
 * You have created a `ShipwrightBuild` resource.
+* You have installed the `oc` CLI.
+* You have installed the `shp` CLI, if needed.
 
 .Procedure
 
 . Create a `Build` resource and apply it to the {ocp-product-title} cluster:
 +
+.Using `oc` CLI
```

Comment:
I do not think this heading is very descriptive. Consider:

> Example of using X to Y.

Global change for other headings.

Reply:
@dfitzmau, I first thought of changing this label to `Using oc CLI to create a Build resource`. But, i think this would make the info more redundant. We have already stated the same in the previous step. The labels over here are just a point of segregation and users just have to decide which CLI they want to use to run a particular command.